tests/test_datastore_utils.py

```diff
@@ -98,6 +100,84 @@ def test_create_alembic_version_multiple_rows():
         create_alembic_version_table(ds.engine, ds.db_type)
 
 
+def test_check_migration_version_new():
+    # Create a new alembic version database - function should pass
+    DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+
+
+def test_check_migration_version_is_found():
+    # Create a new alembic version database - function should pass
+    ds = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+    revision = ["version_id", "test_version_id"]
+
+    # Run once to create the table and stamp latest version
+    create_alembic_version_table(ds.engine, ds.db_type)
+
+    with ds.engine.connect() as connection:
+        connection.execute("UPDATE alembic_version SET version_num = 'version_id';")
+        ds.check_migration_version(revision)
+
+
+def test_check_migration_version_no_revisions():
+    # Call the function with no revisions in the list - function should call sys.exit(1)
+    ds = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+    revisions = []
+
+    with ds.engine.connect():
+        with pytest.raises(SystemExit) as exit_exception_e:
+            temp_output = StringIO()
+            with redirect_stdout(temp_output):
+                ds.check_migration_version(revisions)
+        output = temp_output.getvalue()
+        assert "ERROR: Expected list of known revisions is 0." in output
```

Comment:
Thinking about it, this error message isn't quite right. How about `...is empty` rather than `...is 0`.

Reply:
(I'm aware that this will need changing in the main function too, as well as here)